### ISSUE TITLE
Remove blockfilterindex, peerblockfilters, peerbloomfilters, rpcworkqueue from BIN_ARGS in exports.sh, move them to umbrel-bitcoin.conf

### ISSUE THREAD

**Issue by yuanoook** [getumbrel/umbrel-apps#506]
https://github.com/getumbrel/umbrel-apps/blob/9d39dd6ca428ecc528d54b7cc9f8658bd582403b/bitcoin/exports.sh#L109

I found it's interesting that I update bitcoin.conf and it didn't work.

and the log

```log
2023-04-22T02:19:32Z Config file arg: blockfilterindex="0"
...
2023-04-22T02:19:32Z Command-line arg: blockfilterindex="1"
2023-04-22T02:19:32Z Command-line arg: peerblockfilters="1"
2023-04-22T02:19:32Z Command-line arg: peerbloomfilters="1"
```

I asked GPT-3.5, it says:

It's possible that the developers of Umbrel have chosen to put this configuration line in exports.sh for a specific reason related to their platform or their desired user experience. However, it might be more intuitive to put this configuration line in umbrel-bitcoin.conf instead. If you believe that it would be more appropriate to have this line in umbrel-bitcoin.conf, you could consider reaching out to the Umbrel development team and submitting a feature request or suggestion.

**Comment by JimSB** [comment]
Hey @yuanoook You can reference "Locating the bitcoin.conf File" in the Official FAQ [here](~/umbrel/app-data/bitcoin/data/bitcoin/bitcoin.conf)

You can ignore umbrel-bitcoin.conf it shouldn't be edited

any settings not in the UI you can add to your own bitcoin.conf via
`sudo nano ~/umbrel/app-data/bitcoin/data/bitcoin/bitcoin.conf`

**Comment by nmfretz** [comment]
Thanks for this suggestion @yuanoook. This has now been implemented in the Bitcoin Node app as of 25.1

Closed by https://github.com/getumbrel/umbrel-apps/commit/18568ba1348c9d87f4b84bf407c1982f689d9adf
